################## GOAL ####################
JointTrajectoryPoint[] points               # Joint state at a particular time on the trajectory
float64 delay 0.0                           # How long before the robot should start moving
int32 polynomial_order 3                    # Degrees of freedom in the interpolating polynomial
---
################ RESULT ####################
Statistics[] position_error                 # Contains information on mean, variance, min, max
int32 successful                            # Error code
int32 SUCCESSFUL = 1                        # As it says on the label
int32 INVALID_ARGUMENTS = -1                # Bad input arguments
int32 OTHER = -2                            # Something else went wrong
string message                              # Explanation of problem
---
################ FEEDBACK ##################
JointState actual                           # Actual state measured from joint sensors
JointState desired                          # Desired state from trajectory generator
JointState error                            # Difference between desired and actual
float64 time_remaining                      # Until completion
